### PR TITLE
Some basic updates

### PR DESCRIPTION
**Pull request by KadenZipfel**
Typos, broken links, clarification, etc..

### CODE REVIEW

File: docs/bug_bounty_list.md [hunk 1]
```diff
@@ -1,15 +1,11 @@
 
 The following are ongoing bug bounty programs, either focused on, or including smart contracts in their scope. Issues and PRs are welcome to add new bounties, or remove those which are no longer active.
 
-* [0xProject](https://0xproject.com/wiki#Bug-Bounty): Launched September 2018, Max payout $100k
 * [Airswap](https://medium.com/fluidity/smart-contracts-and-bug-bounty-ad75733eb53f)
 * [Augur](https://www.augur.net/bounty/): Launched October 2018, Max payout $250k
-* [Aragon](https://wiki.aragon.org/dev/bug_bounty/): Launched October 2018, Max payout $250k
```
Comment (sohkai) [inline docs/bug_bounty_list.md]: This is now at https://wiki.aragon.org/association/security/bug_bounty/.

We will also be soon announcing another security audit for the Aragon Network, hosted at a similar location.

Comment (KadenZipfel) [reply docs/bug_bounty_list.md]: Thanks, will update this.

File: docs/bug_bounty_list.md [hunk 2]
```diff
@@ -1,15 +1,12 @@
...
 * [Airswap](https://medium.com/fluidity/smart-contracts-and-bug-bounty-ad75733eb53f)
+* [Aragon](https://wiki.aragon.org/association/security/bug_bounty/)
 * [Augur](https://www.augur.net/bounty/): Launched October 2018, Max payout $250k
-* [Aragon](https://wiki.aragon.org/dev/bug_bounty/): Launched October 2018, Max payout $250k
 * [BrickBlock](https://blog.brickblock.io/join-the-brickblock-bug-bounty-program-7b431f2bcc02): Launched September 2018, Max payout 100 ETH
 * [Colony.io](https://blog.colony.io/announcing-the-colony-network-bug-bounty-f44cabaca9a3/)
 * [Ethereum Foundation](https://bounty.ethereum.org/#bounty-scope): Has a large scope, including clients, Solidity and Vyper, and more.
 * [Etherscan.io](https://etherscan.io/bugbounty)
 * [Gitcoin Bounties](https://gitcoin.co/explorer): Bounty-based collaboration tool
-* [MelonPort](https://melonport.com/bug-bounty)
```
Comment (shayanb) [inline docs/bug_bounty_list.md]: New url: `https://melonprotocol.com/docs/protocol-security/`

File: docs/bug_bounty_list.md [hunk 3]
```diff
@@ -1,15 +1,12 @@
 
 The following are ongoing bug bounty programs, either focused on, or including smart contracts in their scope. Issues and PRs are welcome to add new bounties, or remove those which are no longer active.
 
-* [0xProject](https://0xproject.com/wiki#Bug-Bounty): Launched September 2018, Max payout $100k
```
Comment (shayanb) [inline docs/bug_bounty_list.md]: New url: `https://0x.org/docs/guides/bug-bounty-program#rewards`

**Comment by shayanb** [comment]
Thanks @KadenZipfel .

**Pull request merged**